wiki/computing/aws-quickstart.md

```diff
@@ -0,0 +1,73 @@
+3.  Select the version that matches your operating system (Ubuntu 16.04 / Ubuntu 18.04)
+4.  Select the instance type. If you created a new account on AWS, you will be eligible for free usage of machines in the free tier range. T2.micro which falls under this category can be used to get familiar with AWS. To know more about the free-tier visit <https://aws.amazon.com/free/free-tier-faqs/>
+5.  After selecting your instance, click configure Instance and dd torage.
+6.  Depending on your requirement, select the amount of storage required. The first 30GB of storage is free under the one year eligibility.
+7.  Continue pressing next until you see ‘Security Groups’.
+    Here you will define the open ports for your machine. By default, port 22 will be open for you to ssh into your machine. However, you will have to define certain rules if you want to host different applications.
+8.  Continue pressing ext until you see the review page.
+9.  Launch Instance and select keypair. If you have previously generated a keypair, you can use the same file to access different machines.
+10. To check your instances, click on Instances in the left sidebar.
+
+
+**Now, to login to your instance -**
+1.  Go to your terminal and login to your instance with the following command:\
+    `ssh -i keyPair.pem -L 8000:localhost:8888 ubuntu@instance`
+2.  You may need to set permissions for your key file. This can be done using chmod 400 keyPair.pem
+
+Instead of having to write this huge command in your terminal, you can edit your ssh config file and use an alias to log in to your remote machine. For mac users the config file can be found in **/Users/user_name/.ssh/config**
+
+The file should include\
+*Host alias*\
+*HostName remote_machine_ip*\
+*IdentityFile path_to_keyfile/keyPair.pem*\
+*User ubuntu*
+
+## Using TMUX / SCREEN
+While using ssh for remote access, your connections may be terminated if there is no activity for a long period of time. While training large models, this may be a problem. As your connection is piped through ssh, once the machine is left idle, the connection breaks causing all running applications / processes to terminate. To avoid this we can use Tmux or Screen.
+
+TMUX​ is a ​terminal multiplexer​ for ​Unix-like​ ​operating systems​. It allows multiple ​terminal sessions to be accessed simultaneously in a single window. It is useful for running more than one ​command-line​ program at the same time. It can also be used to detach ​processes from their controlling terminals, allowing ​SSH​ sessions to remain active without being visible.
+*- Wikipedia*
+
+Here are a few important references in this regard
+1.  <https://www.hamvocke.com/blog/a-quick-and-easy-guide-to-tmux/>
+2.  <https://tmuxcheatsheet.com/>
```

Comment:
These need to be in the `[title](url)` format. Jekyll has poor handling for HTML-esque formats.